app/models/order.rb

```diff
@@ -2,7 +2,5 @@ class Order < ApplicationRecord
   belongs_to :user
   validates :title, presence: true, length: { minimum: 2 }
   validates :body, presence: true, length: { minimum: 5 }
+  validates :check_out, presence: { message: "must be in the format XX:XX" }
```

Comment:
```suggestion
  validates :check_out, presence: { message: "required and must be in the format XX:XX" }
```
It could be not obvious that it's required